packages/backend/src/services/CsvService/CsvService.ts

```diff
@@ -377,13 +381,23 @@ export class CsvService {
             });
             const s3Url = await this.s3Service.uploadCsv(csvContent, fileId);
 
-            await fsPromise.unlink(`/tmp/${fileId}`);
-
-            return { filename: `${chart.name}`, path: s3Url };
+            // Delete local file in 10 minutes, we could still read from the local file to upload to google sheets
+            setTimeout(async () => {
+                await fsPromise.unlink(`/tmp/${fileId}`);
```

Comment:
mm, yeah... this method is called on a separate scheduler job(handle) , so it is not easy to know if a target is going to be using this file to read. (on gdrive we read from file and upload the csvcontent) . 

I don't want to return the `csvResults` on this method becuase that could cause OOM errors